inventory/components/InventoryTable.tsx

```
@@ -0,0 +1,281 @@
+            {/* Table */}
+            <div className="overflow-x-auto border border-gray-300 dark:border-gray-600 rounded-lg">
+                <table className="min-w-full border-collapse">
+                    <thead className="bg-gray-50 dark:bg-gray-800">
+                        {table.getHeaderGroups().map((headerGroup) => (
+                            <tr key={headerGroup.id}>
+                                {headerGroup.headers.map((header) => (
+                                    <th
+                                        key={header.id}
+                                        className="px-6 py-3 text-left text-xs font-medium text-gray-500 dark:text-gray-400 uppercase tracking-wider cursor-pointer hover:bg-gray-100 dark:hover:bg-gray-700 border-b border-r border-gray-300 dark:border-gray-600"
+                                        onClick={header.column.getToggleSortingHandler()}
+                                    >
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)